postgresqldatabase.go

```diff
@@ -0,0 +1,283 @@
+package naffka
+
+import (
+	"database/sql"
+	"sync"
+	"time"
+)
+
+const postgresqlSchema = `
+CREATE SEQUENCE IF NOT EXISTS naffka_topic_nid_seq;
+CREATE TABLE IF NOT EXISTS naffka_topics (
+	topic_name TEXT PRIMARY KEY,
```

Comment:
This is a table to lookup topic_name to topic_nid, we don't actually need to go the other way.